Tests/Svg.UnitTests/ImageComparisonTest.cs

```diff
@@ -13,6 +15,32 @@ namespace Svg.UnitTests
     public class ImageComparisonTest
     {
         public TestContext TestContext { get; set; }
+#if NETCORE
+        private static string _basePath = null;
+        private static string GetSuiteTestsFolder
```

Comment:
Wouldn't this also work for the .NET case?

Reply:
It may work - I'll perform the tests tomorrow.

Reply:
That would be nice - I  cobbled that test code together to work somehow, but it is not very robust and I don't really like it.